core/arm/demoDashboard.json

```diff
@@ -70,7 +70,12 @@
     "web-farm-name": "[concat(variables('prefix'), 'webplan', variables('unique'))]",
     "web-site-name": "[concat(variables('prefix'), 'web', variables('unique'))]",
     "web-farm-id": "[resourceId('Microsoft.Web/serverfarms', variables('web-farm-name'))]",
-    "web-site-id": "[resourceId('Microsoft.Web/sites', variables('web-site-name'))]"
+    "web-site-id": "[resourceId('Microsoft.Web/sites', variables('web-site-name'))]",
```

Comment:
can you move ServiceBus into a separate ARM template? 
that way it will be easier for us to remove it if this doesn't work for whatever reason